instrumentation/akka-http-core-2.11_10.0.11/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'scala'
 
 jar {
-    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.akka-http-core-10.0.11' }
+    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.akka-http-core-2.11_10.0.11' }
```

Comment:
Changing the package name is OK, but we can't change the Implementation-Title as it may be in user configs to disable the instrumentation module.

Reply:
@tspring Thanks for letting me know.  I changed the Implementation-Title back to the old version for the scala_2.11 module.